### PR TITLE
Courses: fix middle school page

### PR DESCRIPTION
**Pull request by breville**
The middle-school markdown page had a very subtle existing issue - a space at the beginning of a line - which caused the markdown parsing to fail after adding the link to the new courses page.  

(Also I missed this when eyeballing the changed page, probably because I was scrolling all the way to the bottom to see the new link.)

This reenables activation of the new course explorer page at /educate/curriculum/courses.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Presumably this has been thoroughly tested locally?

File: pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ For Algebra Classrooms
 
 [/solid-block-header]
 
-Code.org has partnered with [Bootstrap](http://www.bootstrapworld.org) to adapt their algebra aligned curriculum to a block based course which teaches algebraic and geometric concepts through computer programming. The lessons focus on concepts like order of operations, the Cartesian plane, function composition and definition, and solving word problems - all within the context of video game design. [Computer Science in Algebra Course A](/educate/algebra) is a ~10 hour, block based course available from Code.org. Or visit Bootstrap to explore their longer [Bootstrap:1](http://www.bootstrapworld.org/materials/spring2016/index.shtml) and [Bootstrap:2](http://www.bootstrapworld.org/materials/spring2016/index.shtml) courses which teach more mathematical and programming concepts. By shifting classwork from abstract pencil-and-paper problems to a series of relevant programming problems, these courses demonstrate how algebra applies in the real world, using an exciting, hands-on approach to create something cool.
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:78]: I don't see where this space is?

File: pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ Computer Science in Science
 
 <i>Note: Starting in the 2017-2018 school year, Code.org will not be offering professional development for this course.</i>
 
- </div>
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:60]: @caley This was the extra space that caused the parsing issue.

File: pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:78]: Oh, and my editor removed a space at the end of this line, which should be innocuous.  You can see it if you highlight the text, but for some reason it hasn't highlighted with a darker block like it normally does in the github UI.

Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/educate/curriculum/middle-school.md:78]: Okay I think this is fine

File: pegasus/sites.v3/code.org/public/educate/curriculum/courses.haml [hunk 3]
```diff
@@ -1,6 +1,5 @@
 ---
 title: Courses
-theme: responsive
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/educate/curriculum/courses.haml:3]: was it required to remove responsiveness on this page?

Comment (caleybrock) [reply pegasus/sites.v3/code.org/public/educate/curriculum/courses.haml:3]: oh I see this is the page you built, never mind.

Comment (breville) [reply pegasus/sites.v3/code.org/public/educate/curriculum/courses.haml:3]: Yes, this is one of our English-only, non-responsive pages.  The course table is not designed to be responsive at this stage.

**Comment by tanyaparker** [comment]
Sorry I didn't test it locally last time. Tested the touched pages lgtm.

**Comment by breville** [comment]
Also tested on Chrome, Safari & Firefox for all affected pages.

**Pull request merged**